cloud_utils/cache/stores/redis.py

```diff
@@ -52,15 +52,16 @@ async def get_item(key: str):
             raise KeyError
 
     async def set_item(key: str, value):
+        ttl_value = ttl(value)
```

Comment:
Why do we want the ttl to be dependent on the value? seems a little odd to me :)